### DISCUSSION TITLE
how can get all policy's and manage them from the go SDK  ??

### DISCUSSION THREAD

**Discussion by aviscode** [minio/minio#13248] [Q&A] [answered]
how can get all policies and manage them from the go SDK? 
and if yes how?

**Comment by klauspost** [comment, marked as answer]
ListBuckets and then GetBucketPolicy/SetBucketPolicy?

**Reply by aviscode** [reply]
@klauspost 
no 
I need all IAM policies, not just the ones assigned to a bucket

**Reply by vadmeste** [reply]
> I need all IAM policies, not just the ones assigned to a bucket

Are you talking about policies that you manage using mc ? `mc admin policy add/list/..` ?

**Reply by aviscode** [reply]
yes, 
but i will like to access it from the code like to write my code and handle the iam polices

**Reply by vadmeste** [reply]
You need to use github.com/minio/madmin-go for that.
